core/project.go

```diff
@@ -169,3 +177,22 @@ func (t *Timetrace) editorFromEnvironment() string {
 
 	return defaultEditor
 }
+
+func (t *Timetrace) parentExists(project Project) error {
+	allP, err := t.ListProjects()
+	if err != nil {
+		return err
+	}
+
+	found := false
+	for _, p := range allP {
+		if p.Key == project.Parent() {
+			found = true
+			break
+		}
+	}
+	if !found {
+		return ErrParentlessModule
+	}
+	return nil
```

Comment:
I think this could even be simplified:

```go
for _, p := range allP {
	if p.Key == project.Parent() {
		return nil
	}
}

return ErrParentlessModule
```